peakdet/utils.py

```diff
@@ -8,6 +8,8 @@
 import inspect
 import numpy as np
 from peakdet import physio
+from loguru import logger
+import sys
```

Comment:
`sys` should go after `inspect` and before `numpy`. `loguru` should go before `peakdet` and `numpy`